src/ansible/roles/packages/tasks/Fedora.yml

```diff
@@ -233,11 +233,32 @@
 
 - name: Install packages for IPA base image
   block:
+  # With additional repos enabled on c8s, installing IPA fails with
+  # Error: No available modular metadata for modular package
+  # 'jaxb-fi-2.1.0-1.module_el8+321+a8d35a21.noarch'
+  - name: Disable crb/buildroot repos due to CentOS stream 8 error
```

Comment:
Do we still build C8S container? Is it ever possible and what for?

Reply:
It was last built successfully 2 months ago according to https://quay.io/repository/sssd/ci-base-ldap?tab=tags - I don't know if it is being used at all.

Reply:
IIUC, c8s repos aren't up-to-date even if available.
We don't use it even in sssd-2-9-4 branch: https://github.com/SSSD/sssd/blob/sssd-2-9-4/contrib/ci/get-matrix.py
I would better stop building c8s container and removed all related code.

Reply:
Okay would you prefer to remove 'c8s' and debian as part of this PR also? CC @pbrezina

Reply:
I added a new commit to drop c8s, let's wait for CI results.

Reply:
centos8 is removed now with CI passing.